core/spec/models/spree/variant_spec.rb

```diff
@@ -812,11 +812,27 @@
   end
 
   describe '#gallery' do
-    let(:product) { Spree::Variant.new }
-    subject { product.gallery }
+    let(:variant) { build_stubbed(:variant) }
+    subject { variant.gallery }
 
     it 'responds to #images' do
       expect(subject).to respond_to(:images)
     end
+
+    context "when variant.images is empty array" do
+      let(:product) { create(:product) }
+      let(:variant) { create(:variant, product: product) }
+
+      it 'should fallback to variant.product.master.images ' do
```

Comment:
Can we avoid using `should` in favor of `fallbacks` here? Also, there's an extra space after `variant.product.master.images`.

Reply:
Done